WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/lists/sections/BaseStatsUseCase.kt

```diff
@@ -94,7 +94,7 @@ abstract class BaseStatsUseCase<DOMAIN_MODEL, UI_STATE>(
                 if (!state.cached) {
                     val updatedCachedData = loadCachedData()
                     if (domainModel != updatedCachedData) {
-                        domainModel = updatedCachedData
+                        domainModel = state.model
```

Comment:
Does it mean the whole if{} block is redundant here?!

Reply:
At first glance, the answer is that its not redundant. Not because of my change, but because of the `updateState()`. If the state values are the same, I imagine we don't want to fire a call to `updateState`. But this is where your input will matter as there may be details about fluxc I don't fully understand. Presumably, if we are using a data class, which I believe we do for those models, data from the cache can equal data from the web. Basing this solely off the [docs](https://kotlinlang.org/docs/data-classes.html). I could be wrong. Your thoughts 😄 ?

Reply:
FluxC is also long before my time!

Wdyt? @irfano

Reply:
You know what. I also just realized that I see why you are saying that. `domainModel != updatedCachedData` is confusing. It should be `domainModel != state`. Does that make more sense @ravishanker ?